attributes/security.rb

```diff
@@ -65,6 +65,18 @@
     end
   end
 
+  # Add cdap group to core-site.xml's hadoop.proxyuser.hive.groups
+  if node['hadoop'].key?('core_site') && node['hadoop']['core_site'].key?('hadoop.proxyuser.hive.groups')
+    arr = node['hadoop']['core_site']['hadoop.proxyuser.hive.groups'].split(',')
+    group = 'cdap'
+    unless arr.include?(group)
+      arr += [group]
+      default['hadoop']['core_site']['hadoop.proxyuser.hive.groups'] = arr.join(',')
+    end
+  else
+    default['hadoop']['core_site']['hadoop.proxyuser.hive.groups'] = 'cdap,hadoop'
```

Comment:
Also, the default set in caskdata/hadoop_wrapper_cookbook#51 has hive in that list, so we'll append to it in practice. This is for people not necessarily running Hive to not have their security mucked with by the cookbook.